wire-ios-request-strategy/Sources/E2EIdentity/E2EIManager.swift

```diff
@@ -0,0 +1,107 @@
+//
+// Wire
+// Copyright (C) 2023 Wire Swiss GmbH
+//
+// This program is free software: you can redistribute it and/or modify
+// it under the terms of the GNU General Public License as published by
+// the Free Software Foundation, either version 3 of the License, or
+// (at your option) any later version.
+//
+// This program is distributed in the hope that it will be useful,
+// but WITHOUT ANY WARRANTY; without even the implied warranty of
+// MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+// GNU General Public License for more details.
+//
+// You should have received a copy of the GNU General Public License
+// along with this program. If not, see http://www.gnu.org/licenses/.
+//
+
+import Foundation
+import WireCoreCrypto
+
+public protocol E2EIManagerInterface {
+
+    /// Fetch acme directory for hyperlinks.
+    func loadACMEDirectory() async -> AcmeDirectory?
```

Comment:
Does this need to be part of the interface? Couldn't we make it internal to the implementation since it seems that the result of this method is only used to invoke the other methods in the interface.